src/module-1/cards-list-v1/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="index-style-card.css">
+        <link rel="stylesheet" type="text/css" href="index-cards-list.css">
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=Barlow:wght@200&display=swap');
+        </style>
```

Comment:
I'd like to have this import outside of the HTML
should be moved to css file like `main.css` and imported:
```
<link rel="stylesheet" type="text/css" href="main.css">
```